environment.yml

```diff
@@ -60,9 +60,15 @@ dependencies:
   - multi-mechanize >=1.2.0
   # - readline >=6.2.4 in the standard library
   - simplejson >=3.8.1
-  - tornado >=5.0.0,<6.0.0
+  #- tornado >=5.0.0,<6.0.0
   - typing >=3.6.6
   # Pin OpenSSL to avoid: https://github.com/DIRACGrid/DIRAC/issues/4489
   - openssl <1.1
   - pip:
     - diraccfg
+    # This is a fork of tornado with a patch to allow for configurable iostream
+    # It should eventually be part of DIRACGrid
+    - git+https://github.com/chaen/tornado.git@iostreamConfigurable
+    # This is an extension of Tornado to use M2Crypto
+    # It should eventually be part of DIRACGrid
```

Comment:
```suggestion
    # It should eventually be part of DIRACOS
```
;)

Reply:
youppeee :D . Thanks, changed

Reply:
No what I mean is that the repo shold be part of the DIRACGrid repos :D